### DISCUSSION TITLE
Rest API - Delete realm-level role mappings returns 401

### DISCUSSION THREAD

**Discussion by vtassios** [keycloak/keycloak#8799] [Q&A]
Hi,

I am using keycloak rest API and I am trying to delete realm-level role mappings using the ```DELETE /{realm}/users/{id}/role-mappings/realm``` but I get status error 401.

On Users -> Role Mappings I have effective client roles all the ```real_management``` and ```account```. 
I've also have an auth token on the request header.

I also get cors policy error. 
On any other request works fine. This is the only one that I get cors policy.

My client has AccessType = public, Valid Redirect URIs = /* and WebOrigins = *.
I've also tried with WebOrigin = + but nothing change.

Does anyone know why I got this error on this specific request and how to solve it?

**Comment by 0x0dr1y** [comment]
Having the same issue for like every endpoint
